### DISCUSSION TITLE
How can I show header while using createMaterialBottomTabNavigator() ?

### DISCUSSION THREAD

**Discussion by AhmedSarhan** [callstack/react-native-paper#3048] [Q&A]
when I'm using the regular createBottomTabNavigator() I have a headerShown prop in options to show/hide top header on screens
but while using createMaterialBottomTabNavigator() I don't have such option at all, is there such a prop or something?

**Comment by raajnadar** [comment]
You need to wrap the `createMaterialBottomTabNavigator()` with stack or drawer and enable header with `headerShow` props as `true`.

**Reply by AhmedSarhan** [reply]
I don't understand, can you share a small code sample please that would be much appreciated

**Reply by raajnadar** [reply]
https://stackoverflow.com/questions/66757162/react-navigation-header-above-tab-nav

**Reply by AhmedSarhan** [reply]
yeah I thought of something similar that would work but I don't want a work around, I just want a regular solution like the other package as I'm using both in the same application one for android and the other for ios

**Reply by raajnadar** [reply]
This is not a workaround this is a solution.
